### ISSUE TITLE
Failing Keras import tests on master 

### ISSUE THREAD

**Issue by maxpumperla** [deeplearning4j/deeplearning4j#5492]
Since I last checked 2 weeks ago, 5 tests are failing, which was not the case before. Tests in question:

```
  KerasModelEndToEndTest.importCnn1d:317->importSequentialModelH5Test:448->importSequentialModelH5Test:464 » ND4JIllegalState
  KerasModelEndToEndTest.importDcganGenerator:296->importSequentialModelH5Test:448->importSequentialModelH5Test:464 » ND4JIllegalState
  KerasModelEndToEndTest.importDgaClassifier:325->importSequentialModelH5Test:448->importSequentialModelH5Test:464 » ND4JIllegalState
  KerasModelEndToEndTest.importFlatIntoLSTM:333->importFunctionalModelH5Test:429->importFunctionalModelH5Test:444 » ND4JIllegalState
  KerasModelEndToEndTest.importWganGenerator:312->importSequentialModelH5Test:448->importSequentialModelH5Test:464 » ND4JIllegalState
```

All of them fail when setting weights, with the following message:
```
org.nd4j.linalg.exception.ND4JIllegalStateException: X, Y and Z arguments should have the same length for PairwiseTransform
```

2 failures due to batch norm, 2 lstms, 1 dense (!) layer. not sure, @raver119, any idea what may have changed since release? I didn't touch model import since then.

**Comment by AlexDBlack** [comment]
Could be the same underlying issue, looking at the stack trace:
https://github.com/deeplearning4j/deeplearning4j/issues/5484

**Comment by raver119** [comment]
Now we check if executed both pairwise transform operands have the same size.
Because if no - people eventually get segfaults.

**Comment by AlexDBlack** [comment]
I'll take a look at this today and isolate it. Will keep you posted.

**Comment by AlexDBlack** [comment]
I've just merged some improvements to the op executioner error messages.
These 4 failures look legitimate (i.e., keras import side issues - they should have been failing before but weren't due to insufficient validation in ND4J).

```
org.nd4j.linalg.exception.ND4JIllegalStateException: X, Y and Z arguments should have the same length for PairwiseTransform. x: length 7, shape [1, 7]; y: 128, shape [1, 128]; z: 7, shape [1, 7]

	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:754)
	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:121)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.assign(BaseNDArray.java:1454)
	at org.deeplearning4j.nn.layers.BaseLayer.setParam(BaseLayer.java:180)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.copyWeightsToLayer(KerasLayer.java:297)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.copyWeightsToModel(KerasModelUtils.java:76)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.getMultiLayerNetwork(KerasSequentialModel.java:234)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.getMultiLayerNetwork(KerasSequentialModel.java:221)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importSequentialModelH5Test(KerasModelEndToEndTest.java:460)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importSequentialModelH5Test(KerasModelEndToEndTest.java:444)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importWganGenerator(KerasModelEndToEndTest.java:308)
```

```
org.nd4j.linalg.exception.ND4JIllegalStateException: X, Y and Z arguments should have the same length for PairwiseTransform. x: length 5000, shape [100, 50]; y: 310000, shape [6200, 50]; z: 5000, shape [100, 50]

	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:754)
	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:121)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.assign(BaseNDArray.java:1454)
	at org.deeplearning4j.nn.layers.BaseLayer.setParam(BaseLayer.java:180)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.copyWeightsToLayer(KerasLayer.java:297)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.copyWeightsToModel(KerasModelUtils.java:76)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.getMultiLayerNetwork(KerasSequentialModel.java:234)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.getMultiLayerNetwork(KerasSequentialModel.java:221)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importSequentialModelH5Test(KerasModelEndToEndTest.java:460)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importSequentialModelH5Test(KerasModelEndToEndTest.java:444)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importDgaClassifier(KerasModelEndToEndTest.java:321)
```

```
org.nd4j.linalg.exception.ND4JIllegalStateException: X, Y and Z arguments should have the same length for PairwiseTransform. x: length 7, shape [1, 7]; y: 128, shape [1, 128]; z: 7, shape [1, 7]

	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:754)
	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:121)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.assign(BaseNDArray.java:1454)
	at org.deeplearning4j.nn.layers.BaseLayer.setParam(BaseLayer.java:180)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.copyWeightsToLayer(KerasLayer.java:297)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.copyWeightsToModel(KerasModelUtils.java:76)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.getMultiLayerNetwork(KerasSequentialModel.java:234)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.getMultiLayerNetwork(KerasSequentialModel.java:221)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importSequentialModelH5Test(KerasModelEndToEndTest.java:460)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importSequentialModelH5Test(KerasModelEndToEndTest.java:444)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importDcganGenerator(KerasModelEndToEndTest.java:292)
```

```
org.nd4j.linalg.exception.ND4JIllegalStateException: X, Y and Z arguments should have the same length for PairwiseTransform. x: length 2560, shape [10, 256]; y: 256, shape [1, 256]; z: 2560, shape [10, 256]

	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:754)
	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.exec(NativeOpExecutioner.java:121)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.assign(BaseNDArray.java:1454)
	at org.deeplearning4j.nn.layers.BaseLayer.setParam(BaseLayer.java:180)
	at org.deeplearning4j.nn.layers.wrapper.BaseWrapperLayer.setParam(BaseWrapperLayer.java:232)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.copyWeightsToLayer(KerasLayer.java:297)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelUtils.copyWeightsToModel(KerasModelUtils.java:76)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.getComputationGraph(KerasModel.java:377)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.getComputationGraph(KerasModel.java:363)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importFunctionalModelH5Test(KerasModelEndToEndTest.java:440)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importFunctionalModelH5Test(KerasModelEndToEndTest.java:425)
	at org.deeplearning4j.nn.modelimport.keras.e2e.KerasModelEndToEndTest.importFlatIntoLSTM(KerasModelEndToEndTest.java:329)
```

**Comment by maxpumperla** [comment]
ok, closing this now. still shocking to me that this silently failed before, but all of them were legitimate issues...
